py-core/app/util/scripts/start_node_snapshot.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
```

Comment:
In all scripts, state your parameters upfront with comments. Then, in the rest of the code, use these named parameters.